core/server/data/importer/importers/data/settings.js

```diff
@@ -54,6 +58,12 @@ class SettingsImporter extends BaseImporter {
 
         _.each(this.dataToImport, function (obj) {
             obj.key = self.legacySettingsKeyValues[obj.key] || obj.key;
+
+            if (obj.key === 'labs' && labsSetting && labsSetting.value) {
```

Comment:
I think we can simply use `obj.value` here?
Other than that the fix looks good and works 👍